set.mm

```diff
@@ -403907,7 +403907,6 @@ in common (for j=1, ... , k).  In contrast to the definition in Aksoy et
     cres cop swrd0lenOLD eqtrd ) BADECFBGHIJEKZBFCLJQZGHBFCRMJZGHCUAUBUCGUAUCUB
     ABCNOPABCST $.
 
-
   ${
     $( TODO-AV: ` H = ( F |` ( 0 ..^ N ) ) ` could be replaced by
                 ` H = ( F prefix N ) ` ! $)
```

Comment:
These two lines are not part of the current PR, so I guess it will be done in a future PR.